src/test/mocks/MockOracle.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {LinkTokenReceiver} from "../utils/LinkTokenReceiver.sol";
```

Comment:
Why do we need to have this in our repo? Do we need it in our repo?

Reply:
If I'm not mistaken, there isn't a `LinkTokenReceiver.sol` contract in `chainlink-brownie-contracts/contracts/src/v0.8` and the `v0.7` version is pinned using `^0.7.0` so unable to use that instead.